theme/accessibility.less

```diff
@@ -0,0 +1,61 @@
+/* Import all components */
+@import 'themes/default/globals/site.variables';
+@import 'themes/pxt/globals/site.variables';
+
+/* Reference import */
+@import (reference) "semantic.less";
+
+/*******************************
+          Accessibility
+*******************************/
+
+*:focus{
+    outline: 2px solid #4D90FE !important;
+}
```

Comment:
Let's specialize this a little more. Doing this for all elements is not a good call, given that some elements could be disabled.
Here's what microsoft.com uses: 

```
a:focus:not([tabindex='-1']), input:not([disabled]):focus:not([tabindex='-1']), select:not([disabled]):focus:not([tabindex='-1']), textarea:not([disabled]):focus:not([tabindex='-1']), button:not([disabled]):focus:not([tabindex='-1']), [tabindex]:focus:not([tabindex='-1']), [contentEditable=true]:focus:not([tabindex='-1'])
```